requirements-development.txt

```diff
@@ -14,4 +14,4 @@ Sphinx
 sphinx_rtd_theme
 tox
 twine
-
+django-filter>=2.0
```

Comment:
let's add this to setup.py as well as test dependency (same way done with django-polymorphic)

Reply:
I couldn't add the dependency to setup.py due to the requirement for 1.1 for Py27 vs. 2.0 for Py3. I don't think there's a way to conditionalize it there. That's why it's in tox.ini. django-polymorphic doesn't require a back-level version for py27.

Reply:
:+1:  Makes sense. Let's leave this out then.